### PR TITLE
Add search command

### PR DESCRIPTION
**Pull request by prawnzyy**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/SearchCommand.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+public class SearchCommand extends Command {
+    public static final String COMMAND_WORD = "search";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all recipes that uses the ingredient within the recipe book "
+            + "Parameters: "
+            + "ingredient name "
+            + "Example: " + COMMAND_WORD + " flour";
+
+    public static final String MESSAGE_SUCCESS = "%$s ingredients listed";
```
Comment (ItsTYtan) [inline src/main/java/seedu/address/logic/commands/SearchCommand.java:25]: Don't know if this needs to be here

**Review by ItsTYtan — COMMENTED** [review]
Seems good except for one thing you might want to remove.

File: src/main/java/seedu/address/logic/commands/SearchCommand.java (hunk 1 above)
Comment (prawnzyy) [reply src/main/java/seedu/address/logic/commands/SearchCommand.java:25]: May i know the reason why you think it need not be there

Comment (dhruvir29) [reply src/main/java/seedu/address/logic/commands/SearchCommand.java:25]: I think it does need to be there though

**Review by dhruvir29 — APPROVED** [review]
LGTM!

**Pull request merged**